VideoLocker/src/main/java/org/edx/mobile/view/custom/CustomSelectView.java

```diff
@@ -114,9 +114,7 @@ public void onClick(View v) {
 
     @Override
     public void onClick(DialogInterface dialog, int which) {
-        checkedItemIndex = which;
-        selectedItem = items.get(which);
-        select(selectedItem);
+        selectItem(which);
```

Comment:
Off topic for this pull request, but I don't see why we're using this custom widget instead of the standard `Spinner` implementation provided by the framework? This seems to be just a combination of `TextView` and `AlertDialog`, and I don't see that it provides anything that's not already possible with the existing widget. I guess it's fine for now, but if we're ever redesigning the sign-up screen, then we should consider refactoring this as well.